src/sgame/components/TurretComponent.cpp

```diff
@@ -125,31 +127,31 @@ bool TurretComponent::MoveHeadToTarget(int timeDelta) {
 		targetReached = false;
 	}
 
-	if (Math::DistanceSq(oldRelativeAimAngles, relativeAimAngles) > 0.0f) {
+	if (glm::distance2(oldRelativeAimAngles, relativeAimAngles) > 0.0f) {
 		turretLogger.Debug(
 			"Aiming. Elapsed: %d ms. Delta: %.2f. Max: %.2f. Old: %s. New: %s. Reached: %s.",
 			timeDelta, deltaAngles, maxAngleChange, oldRelativeAimAngles, relativeAimAngles, targetReached
 		);
 	}
 
 	// TODO: Move gentity_t.buildableAim to BuildableComponent.
-	Vec3 absoluteAimAngles = RelativeAnglesToAbsoluteAngles(relativeAimAngles);
-	absoluteAimAngles.Store(entity.oldEnt->buildableAim);
+	glm::vec3 absoluteAimAngles = RelativeAnglesToAbsoluteAngles(relativeAimAngles);
+	absoluteAimAngles = VEC2GLM(entity.oldEnt->buildableAim);
```

Comment:
This isn't equivalent. You want 
```suggestion
	VectorCopy(absoluteAimAngles, entity.oldEnt->buildableAim);
```